compiler/test/fail_compilation/fail10905.d

```diff
@@ -17,6 +17,6 @@ struct Bar
 
     bool spam() const
     {
-        return x == Foo.y;
```

Comment:
Instead of modifying the test case, it would be better to just move it over to the compilable directory. I assume you are going to implement `!=` too, so you will have to do this sooner or later.

Reply:
I don't see any testing advantage to either way, but this way entails less disruption.